### ISSUE TITLE
watching repos - can we automate this?

### ISSUE THREAD

**Issue by alanswx** [CommerceGov/Policies-and-Guidance#6]
Is there a reason to email the public repos?  What if there was a GitHub DOC watch account that each subaccount added  (as read only?). Then it could watch all the repos automatically.  Otherwise the repo owners wouldn't necessarily know if they forgot to email doc with the link.  Downside is that this DOC account would/could see all private repos as well? I am not sure if this causes any problems.

"After you have published your content to a DOC bureau’s public GitHub repository, you must email mkruger@doc.gov with the link to your repository so that the DOC GitHub account can “watch” your repository and be notified of updates. DOC will do this by selecting the “Watch” button at the top of your repository. All DOC bureau public repositories should be “watched” by DOC GitHub administrators so that they appear on the administrators’ dashboards."

**Comment by ianjkalin** [comment]
I think the original intent here was more about monitoring accounts for unauthorized or inappropriate actions.  But even then, I personally don't understand why a program like Github should be actively watched by anyone.  It's not reasonable to do so manually given the diversity and size of documents that will be created.  By analogy, no one is manually watching the Microsoft Word documents I create on a share-drive.  Therefore, I think this whole section will need to be re-worked.

**Comment by mike-charles** [comment]
Agreed, I don't see how they could possibly monitor every single repository.
